lib/project.ex

```diff
@@ -0,0 +1,262 @@
+defmodule Slax.Project do
```

Comment:
Add some docs about the module and functions. I'm not sure (from first glance) what the steps attribute does because because new_project/5 also mentions the different things to create